include/stlink/tools/flash.h

```diff
@@ -26,9 +26,10 @@ struct flash_opts
     uint32_t val;
     size_t flash_size;	/* --flash=n[k][m] */
     int opt;
+    int freq;
```

Comment:
I think an inline comment would be useful here  (equal to `stlink.h`).

Reply:
I will add a comment in the coming commit tomorrow

Reply:
Added

Reply:
Thanks!